internal/commands/build/build.go

```diff
@@ -45,24 +45,25 @@ type buildOptions struct {
 func Cmd(dockerCli command.Cli) *cobra.Command {
 	var opts buildOptions
 	cmd := &cobra.Command{
-		Use:     "build [OPTIONS] [CONTEXT_PATH]",
-		Short:   "Build service images for the application",
-		Example: `$ docker app build --tag my/app:1.0.0 .`,
-		Args:    cli.ExactArgs(1),
+		Use:   "build [OPTIONS] [CONTEXT_PATH]",
```

Comment:
docker build help is the following:
`docker build [OPTIONS] PATH`.
So should we have the same synopsis?
`build [OPTIONS] PATH` ?

Reply:
In the context of docker app, "PATH" is not as obvious as for `docker build`, as we have this x.dockerapp thing. "CONTEXT" might not make much sense but to docker gurus. "BUILD_PATH" ?

Reply:
OK, checked with Chris, I have modified to BUILD_PATH.